data/presets/presets/building/apartments.json

```diff
@@ -1,5 +1,9 @@
 {
     "icon": "maki-building",
+    "fields": [
+        "{building}",
+        "flats"
```

Comment:
Since the preset file was moved to a folder, the new field ID needs to be `building/flats`.

Reply:
Hi Quincy, thanks for your patience :)
Changes have been made as requested